src/khoj/interface/web/chat.html

```diff
@@ -372,7 +408,47 @@
                 copyIcon.classList.add("copy-icon");
                 copyButton.appendChild(copyIcon);
                 copyButton.addEventListener('click', createCopyParentText(message));
-                element.append(copyButton);
+
+                var thumbsUpButtons = document.querySelectorAll('.thumbs-up-button');
+                var thumbsDownButtons = document.querySelectorAll('.thumbs-down-button');
+
+                //clear out old feedback buttons as we can't access this data anymore
+                thumbsUpButtons.forEach(function(element) {
+                    element.parentNode.removeChild(element);
+                });
+                thumbsDownButtons.forEach(function(element) {
+                    element.parentNode.removeChild(element);
+                });
```

Comment:
Why do you need to do this? Each message should be able to keep its feedback buttons. The data should still be accessible for the user/AI query.